test/Atc.Tests/Helpers/NetworkInformationHelperTests.cs

```diff
@@ -8,6 +9,15 @@ public class NetworkInformationHelperTests
     public void HasConnection()
         => Assert.True(NetworkInformationHelper.HasConnection());
 
+    [Fact]
+    public void HasHttpConnection()
+        => Assert.True(NetworkInformationHelper.HasHttpConnection());
+
+    [Theory]
+    [InlineData("https://www.google.com/")]
```

Comment:
Shouldn't this test contain a "bad" case as well? Otherwise a "return true" implementation would be perfectly valid. ;-)